crates/primitives/src/specification.rs

```diff
@@ -62,9 +62,10 @@ pub enum SpecId {
     KATLA = 17,
     HEKLA = 18,
     ONTAKE = 19,
-    CANCUN = 20,
-    PRAGUE = 21,
-    OSAKA = 22,
+    PACAYA = 20,
```

Comment:
I would expect it to be safe that we change spec id numbering, hope that it won't affect forkids and you won't need to fix all tests related to this :)

Reply:
Yeah it's bit weird that we are doing  it this way. 
Maybe better solution would be to control this via feature flags, as it is done in other parts of code?